examples/digital_fingerprinting/production/conda_env.yml

```diff
@@ -30,4 +30,4 @@ dependencies:
     - mlflow>=2.2.1,<3
     - nodejs=17.4.0
     - papermill
-    - s3fs==2022.8.2
+    - s3fs==2023.6
```

Comment:
Lets loosen the version restriction here. Either `s3fs>=2023.6` or `s3fs=2023.6` to allow for hotfixes should work. Pinning to the exact version in conda can be problematic